src/encoder.rs

```diff
@@ -2131,17 +2135,25 @@ fn encode_tile(sequence: &mut Sequence, fi: &FrameInvariants, fs: &mut FrameStat
             cw.bc.cdef_coded = false;
             cw.bc.code_deltas = fi.delta_q_present;
 
+            // Do subsampled ME
+            let mut pmvs: [Option<MotionVector>; REF_FRAMES] = [None; REF_FRAMES];
+            for i in 0..INTER_REFS_PER_FRAME {
+                let r = fi.ref_frames[i] as usize;
+                if pmvs[r].is_none() {
+                    pmvs[r] = estimate_motion_ss4(fi, fs, r, &bo);
```

Comment:
So, this is 1/4 resolution ME for a SB, I think?

Reply:
Also, not for this patch but I'd like to remind use that this approach of hierarchical ME starting from SB level can be  still benefited from using the MVs from already encoded neighbors as filed in #594.